src/Microsoft.Health.Fhir.Core/Configs/ExportJobConfiguration.cs

```diff
@@ -3,13 +3,21 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
+
 namespace Microsoft.Health.Fhir.Core.Configs
 {
-    public class ExportConfiguration
+    public class ExportJobConfiguration
     {
         /// <summary>
         /// Determines whether export is enabled or not.
         /// </summary>
         public bool Enabled { get; set; }
+
+        public ushort MaximumNumberOfConcurrentJobsAllowed { get; set; } = 1;
+
+        public TimeSpan JobHeartbeatTimeoutThreshold { get; set; } = TimeSpan.FromMinutes(10);
```

Comment:
We might want to make this setting something like an `int` representing seconds. I'm not sure how common a string representation of `TimeSpan` is in json.

Reply:
Originally I had these defined as `JobHeartbeatTimeoutThresholdInSeconds` and had the value being a number but decided to make it into `TimeSpan` since I felt it gives better flexibility and is more readable. 

I didn't see any downside of using `TimeSpan` but I am open to change.

Reply:
My opinion is that a large number representing seconds is not that readable. I would vote for the TimeSpan format, even if is is harder to author.